src/sentry/integrations/utils/code_mapping.py

```diff
@@ -142,20 +136,20 @@ def _get_code_mapping_source_path(self, src_file: str, frame_filename: FrameFile
 
     def _generate_code_mapping_from_tree(
         self,
-        repo_full_name: str,
+        repo_tree: RepoTree,
         frame_filename: FrameFilename,
     ) -> List[CodeMapping]:
         matched_files = [
             src_path
-            for src_path in self.trees[repo_full_name].files
```

Comment:
The bug we're seeing is because over here we had a dictionary and we should have been calling `["files"]`. I still can't figure out why the tests did not trigger this issue.

I have been able to reproduce the issue because I did a real integration run from a script I've written against my GH org.